object-describer.js

```diff
@@ -5,16 +5,6 @@ try { angular.module("openshiftUI") } catch(e) { angular.module("openshiftUI", [
 angular.module("openshiftUI").requires.push("kubernetesUI");
 
 angular.module('openshiftUI')
-.filter('imageRepoReference', function(){
```

Comment:
Keep this, and make it do something close to https://github.com/openshift/origin/blob/master/pkg/build/generator/generator.go#L275-L325

```
if (objectRef.kind == "DockerImage") {
  return objectRef.name;
}
var ns = objectRef.namespace || "";
ns = ns == "" ? ns : ns + "/";
var ref = ns + objectRef.name;
return ref;
```
tag is embedded in the name now in the case of `kind==ImageStreamTag`, so I don't think we need it passed in separately